demos/github-demo/src/isograph-components/home_page.tsx

```diff
@@ -17,7 +16,7 @@ export const home_page_component = iso<
   HomePageComponentParams,
   ReturnType<typeof HomePageComponent>
 >`
-  Query.home_page_component($first: Int!) @component {
+  field Query.home_page_component($first: Int!) @component {
```

Comment:
Looks great. If you run `npm run build-demos` you'll find other places you missed

Reply:
Fixed with [`99ee3eb` (#4)](https://github.com/isographlabs/isograph/pull/4/commits/99ee3eb52baeddc081badcdc8866d878dc4e97ad)